code.go

```diff
@@ -20,6 +20,15 @@ import (
 	"github.com/pkg/errors"
 )
 
+var (
+	// To detect a line containing low-level requirements
+	reLLRReferenceLine = regexp.MustCompile(`^[ \*\/]*(?:@|\\)llr (?:REQ-\w+-\w+-\d+[, ]*)+$`)
```

Comment:
Minor nit. Shouldn't reqtraq support any number of spaces between `\llr` and the requirement ID?